terraform/config_production.tfvars

```diff
@@ -3,5 +3,5 @@ environment   = "production"
 cpu           = 1024
 memory        = 2048
 service_count = 3
-min_capacity  = 2
```

Comment:
Aren't people still using the production one whilst we bed in with the merged dev/staging versions?

Reply:
If it's working in staging it'll work in production most likely.   If we're confident enough to merge we should be confident enough to run in prod.